src/thorlabs_cube/driver/tcube/tsc.py

```diff
@@ -0,0 +1,360 @@
+import struct as st
+
+from thorlabs_cube.driver.base import _Cube
+from thorlabs_cube.driver.message import MGMSG, Message, MsgError
+
+_CHANNEL = 0x01
+_RESERVED = 0x00
```

Comment:
```suggestion
_CHANNEL: int = 0x01
_RESERVED: int = 0x00
```
Add type annotations to constants.

Now that I've seen this exact code in so many files, would it be better to make these constants class attributes of the `_Cube` base class?

Reply:
That's actually a really good idea, will be implemented for your next review!